ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -169,6 +181,17 @@ open class ContactView: UIView {
         ]
     }
 
+    private func pressedStateOverlayLayoutConstraints() -> [NSLayoutConstraint] {
+        return [
+            pressedStateOverlay.widthAnchor.constraint(equalTo: avatarView.widthAnchor),
+            pressedStateOverlay.heightAnchor.constraint(equalTo: avatarView.heightAnchor),
```

Comment:
The width and height anchor are not needed and add extra complexity to the layout. It's important to always use the smallest amount possible of constraints to layout a view. In this case, if the leading, trailing, top and bottom anchors are equal to the super view, then the width and height have to be equal too so we don't need those extra constraints.

Reply:
Understood, thanks for explaining the reasoning.